src/styles/StyleUtils.js

```diff
@@ -310,7 +310,7 @@ function getFontFamilyMonospace({fontStyle, fontWeight}) {
  */
 function getEmojiPickerStyle(isSmallScreenWidth) {
     return {
-        width: isSmallScreenWidth ? '100%' : CONST.EMOJI_PICKER_SIZE,
+        width: isSmallScreenWidth ? '100%' : CONST.EMOJI_PICKER_SIZE.WIDTH,
```

Comment:
We should also apply height to it, otherwise, Picker's height is nondeterministic and it won't be anchored correctly. 
We need that height to place the popover correctly.